src/interfaces/api/community.ts

```diff
@@ -23,7 +23,7 @@ export interface CreateCommunity {
   description?: string;
   icon?: string;
   banner?: string;
-  nsfw: boolean;
+  nsfw?: boolean;
```

Comment:
Why has this become optional?

Reply:
I think we should not use Optionals as much as possible especially in cases like Bools, Vecs

Reply:
I noticed these ones in particular are optional on the back end anyway, since the DB column has a default value. 

There are also a lot of boolean fields in the requests, that aren't DB columns, that could potentially be converted to optionals. I'd probably prefer them that way, but I haven't included converting all those yet.